### ISSUE TITLE
Cannot connect in windows multiplayer

### ISSUE THREAD

**Issue by micronizerbattheend** [Wargus/stratagus#161]
Hello
I recently installed stratagus and wargus 2.4.0 on windows 8. When creating a multiplayer server, clients cannot join (they time out trying to do so); I got the same results when trying to join my own server (localhost) from another running copy of the program in the same machine (that makes me believe it's not a firewall issue).

Thanks

**Comment by timfel** [comment]
If you're joining from the same machine, I think you'd have to pass an argument to the executable to select a different port for the second instance, I think they would conflict otherwise. Just to be sure, did you check the firewall rules?

**Comment by Andrettin** [comment]
You probably need to port-forward:
http://forums.stratagus.com/viewtopic.php?f=26&t=3611

**Comment by Kintobor** [comment]
It might be an idea to use two ports in Stratagus. One for hosting, the other for joining.

**Comment by micronizerbattheend** [comment]
@timfel I did check the firewall rules and disabled windows firewall; however I'm not too familiar with this version of windows so I don't know if I might be missing something else.

I used the -P argument to choose port as suggested in the post linked by @Andrettin but I still can't connect to localhost:port (or typing my ip:port) even if I start two instances with different -P arguments.

Regarding port forwarding, I should point out that I don't have an "internal IP" behind a router and an "external ip"; both of my IP are the same and I don't seem to have problems hosting servers for other games. But for now even if accepting external connections doesn't work I would be content being able to try simulated multiplayer in my own machine. (That would confirm for me that it's a connectivity problem on my end)

Would it be possible (if it's not too much trouble) for anyone to confirm that multiplayer is working ok in this version? If it's my fault I can keep on troubleshooting locally.
